src/BenchmarkDotNet/Configs/DefaultConfig.cs

```diff
@@ -87,5 +88,11 @@ public IEnumerable<IValidator> GetValidators()
         public IEnumerable<HardwareCounter> GetHardwareCounters() => Array.Empty<HardwareCounter>();
 
         public IEnumerable<IFilter> GetFilters() => Array.Empty<IFilter>();
+
+        /// <summary>
+        /// Determines if the current runtime supports threading.
+        /// </summary>
+        /// <remarks>This property returns <c>false</c> when running under WebAssembly, otherwise <c>true</c>.</remarks>
+        public static bool SupportsThreading { get; } = !RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY"));
```

Comment:
Could you please move it to our `RuntimeInformation` class?

https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet/Portability/RuntimeInformation.cs